plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -180,6 +180,7 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
             override val sourceName: String?
                 get() = super<NamedFunctionSignature>.sourceName
         }
+        symbol.valueParameterSymbols
```

Comment:
What does this do?

Reply:
I didn't notice, strange that a warning wasn't issued.